Lesson30/Kubra/my-app/src/components/Exercise10/index.jsx

```diff
@@ -0,0 +1,10 @@
+export const Exercise10 = () => {
+
+    const handleClick = () => {
+        alert('Double-Clicked!!!!')
```

Comment:
If you want to check a double click event, you need to check the event property. Should be like this instead:
```
if (event.detail === 2) {
        alert('Double-Clicked!!!!')
}

```